contracts/src/v0.8/functions/dev/v1_X/accessControl/TermsOfServiceAllowList.sol

```diff
@@ -197,4 +201,13 @@ contract TermsOfServiceAllowList is ITermsOfServiceAllowList, IAccessController,
 
     return blockedSenders;
   }
+
+  function updateFromPrevious() external override {
```

Comment:
nit: Can we call this something slightly more explicit, e.g. `migratePreviousContractList`?